service/tx/service.go

```diff
@@ -98,6 +98,53 @@ func (svc *Service) SendTx(args client.SendTxRequest, reply *client.CreateTxRepl
 	return nil
 }
 
+func (svc *Service) SendPoolTx(args client.SendPoolTxRequest, reply *client.CreateTxReply) error {
```

Comment:
Is this function only for testing? When you try to sign with the from address, the node wallet might not have the private key.